webtau-http/src/main/java/org/testingisdocumenting/webtau/http/config/HttpConfigurations.java

```diff
@@ -47,10 +47,14 @@ public static void remove(HttpConfiguration configuration) {
     }
 
     public static String fullUrl(String url) {
-        if (! enabled.get()) {
+        if (!enabled.get()) {
             return url;
         }
 
+        return fullUrlWithoutEnabledCheck(url);
+    }
+
+    public static String fullUrlWithoutEnabledCheck(String url) {
```

Comment:
might be worth adding a few comments why you'd want to run fullUrl vs fullUrlWithoutEnabledCheck

Reply:
next step would be to separate those two and hopefully there won't be a need. @tsiq-karold did you by any chance created a ticket for this?

Reply:
I haven't.  By those two you mean url and header, right?